### DISCUSSION TITLE
Better *keywords*

### DISCUSSION THREAD

**Discussion by OmiChrome** [jsmsj/GdriveCloneBot#4] [Ideas]
make better coomands  as 

privclone = pr  `or`  priv
pubclone = pu `or`   pubc

also give option to make every commands a bit short like 

size= g            <- should be used as    `g s LINK` 

also there is no way a reason to `clone` in commands , since its just a clone bot not a mirror bot , so whats the point of making commands with specifc with clone word?

**Comment by jsmsj** [comment]
i agree that i don't have the best name for commands. I just named them on the go.

and i'll update these once im done with my exams.

**Reply by OmiChrome** [reply]
exam of ?
